cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -131,7 +140,7 @@ <h1>
                         <div class="exp-name">{{ experiment.name }}</div>
                         <div class="select-experiment font-bold text-2xl" title="Select Experiment" data-accession="{{ experiment.accession_id }}" data-name="{{ experiment.name }}">＋</div>
                     </div>
-                    <div class="description">{{ experiment.description }}</div>
+                    <div class="exp-description">{{ experiment.description }}</div>
```

Comment:
I actually thought the description looked fine as it was. Maybe we can just delete the exp-class remove the class attribute from this div?

Reply:
The description is still black but it turns blue when you hover over it. None of our links with text do this when hovering. Making sure everything is consistent.

Reply:
exp-description was already there. A little confused at what you are saying.

Reply:
oh dang, I think I jammed several thoughts together because I got distracted in the middle of writing that sentence. 

`exp-description` was already defined in the `styles` section, but it wasn't being used anywhere. This `div`'s class was `description`, which wasn't defined anywhere. So you could delete the `exp-description` rule from the page's CSS and delete the `description` class from this `div` and nothing would change. 

Regarding the "blue when you hover" -- it's only blue if the link hasn't been visited. This suggests that there some CSS somewhere that either 1) causes this or 2) blocks it everywhere else. 

Indeed there is! I guessed that the relevant CSS would be in project.css.tw and did a search for ":visited", which shows up twice. Once is as part of a data-table (almost certainly irrelevant to this) and once is as part of a plain `a` tag selector:

https://github.com/ReddyLab/cegs-portal/blob/main/cegs_portal/static/css/project.css.tw#L67 

That doesn't look like it causes things to be blue*, but the `a:hover` above it looks suspicious. If we check what color `text-blue-800` is in the [tailwind documentation](https://tailwindcss.com/docs/text-color) it matches what we see. This means we're probably looking at case (1) that some CSS is causing this blue/purple hover effect. And you think we don't want this to happen here, or anywhere (I agree with you). 

So the solution is to remove this a:hover CSS rule from project.css.tw. Additionally, you can remove the `exp-description` CSS rule from experiment_list.html and the `description` class from the div. Also, be sure to remove the `description` class from the html generated in facet_filter.js.

*The reason we don't see the blue when we hover over visited links because `a:visited` overrides the hover color rule with `color: inherit`, which keeps the color the same as it was before.

Reply:
1. Yes, I remember now. We are using a:hover for location in dna_feature.html page and other pages. Ex. This helps the user know that the location is clickable in the DNA feature profile table. It works well on those pages but does not look the best here on experiment_list page. From what I can recall, I didn't visit all the links and all were still black. Strange. We used the classes for a reason and I believe this may be it--to set a text color.  
3. Yes, I remember adding the hover color rule but I think you addressed the issue.

Reply:
If you want to change how the descriptions work only on this page you can replace the .exp-description definition with 

```css
a.exp-list-content-container-link:hover {
    color: inherit;
}
```

which gets at the root of the issue